From: Andrew Morton <akpm@linux-foundation.org>
Subject: restore-acct_reclaim_writeback-for-folio

Make Mel's "mm/vmscan: throttle reclaim and compaction when too may pages
are isolated" work for folio changes.

Cc: Matthew Wilcox <willy@infradead.org>
Cc: Mel Gorman <mgorman@techsingularity.net>
Cc: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/filemap.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/filemap.c~restore-acct_reclaim_writeback-for-folio
+++ a/mm/filemap.c
@@ -1600,6 +1600,7 @@ void folio_end_writeback(struct folio *f
 
 	smp_mb__after_atomic();
 	folio_wake(folio, PG_writeback);
+	acct_reclaim_writeback(folio_page(folio, 0));
 	folio_put(folio);
 }
 EXPORT_SYMBOL(folio_end_writeback);
_
